python/docs/source/index.rst

```diff
@@ -68,6 +68,15 @@ API Reference
    sbp.tracking
    sbp.utils
 
+Code Documentation
+------------------
+:ref:`modindex`
+
+.. toctree::
+   :maxdepth: 1
+
+   modules.rst
```

Comment:
this is needed to avoid a sphinx warning, which `-W` causes to be an error

Reply:
👍